Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingsList.swift

```diff
@@ -139,44 +190,55 @@ struct OfferingsList: View {
                     #if !os(watchOS)
                     .contextMenu {
                         let rcOffering = responsePaywall.convertToRevenueCatPaywall(with: responseOffering)
-                        self.contextMenu(for: rcOffering)
+                        self.contextMenu(for: rcOffering, responseOfferingID: offeringPaywall.offering.id)
                     }
                     #endif
                 } header: {
                     Text(responseOffering.displayName)
                 }
             }
         }
+        .refreshable {
+            Task { @MainActor in
+                await updateOfferingsAndPaywalls()
+            }
+        }
         .sheet(item: self.$presentedPaywall) { paywall in
             PaywallPresenter(offering: paywall.offering, mode: paywall.mode)
                 .onRestoreCompleted { _ in
                     self.presentedPaywall = nil
                 }
+                .id(presentedPaywall?.hashValue) //FIXME: This should not be required, issue is in Paywallview
```

Comment:
I commented on slack, maybe we can get around this by making `Offering` conform to `ObservableObject` and making the Offering a `StateObject`?

Reply:
As discussed elsewhere, we will return to this outside this PR. Thanks!